### ISSUE TITLE
[BUG] headless: false & executablePath: null in AWS Lambda

### ISSUE THREAD

**Issue by shanedasilva** [alixaxel/chrome-aws-lambda#171]
## Environment
* `chrome-aws-lambda` Version: 5.3.1
* `puppeteer` / `puppeteer-core` Version: 5.4.0
* OS: Mac
* Node.js Version: v14.13.0
* Lambda / GCF Runtime: nodejs10.x

## Expected Behavior
When running the below script in a Lambda environment, I expect headless = true and executablePath to populaete

## Current Behavior
Headless is always false and executablePath is always null.

## Steps to Reproduce

```js
const chromium = require('chrome-aws-lambda');

exports.handler = async (event, context, callback) => {
  let result = null;
  let browser = null;

  try {
    browser = await chromium.puppeteer.launch({
      args: chromium.args,
      defaultViewport: chromium.defaultViewport,
      executablePath: await chromium.executablePath,
      headless: chromium.headless,
      ignoreHTTPSErrors: true,
    });

    let page = await browser.newPage();

    await page.goto(event.url || 'https://example.com');

    result = await page.title();
  } catch (error) {
    return callback(error);
  } finally {
    if (browser !== null) {
      await browser.close();
    }
  }

  return callback(null, result);
};
```

**Comment by alixaxel** [comment]
@shanedasilva I cannot reproduce this, it works as expected for me:

```json
{
  "executablePath": "/tmp/chromium",
  "headless": true,
  "version": "v10.23.0"
}
```

![image](https://user-images.githubusercontent.com/262782/99889189-91305d00-2c4a-11eb-92b4-e852d7f13eec.png)

```js
const chromium = require('chrome-aws-lambda');

exports.handler = async (event, context, callback) => {
  let result = null;
  let browser = null;

  try {
    browser = await chromium.puppeteer.launch({
      args: chromium.args,
      defaultViewport: chromium.defaultViewport,
      executablePath: await chromium.executablePath,
      headless: chromium.headless,
      ignoreHTTPSErrors: true,
    });

    let page = await browser.newPage();

    await page.goto(event.url || 'https://example.com');

    result = await page.title();
  } catch (error) {
    return callback(error);
  } finally {
    if (browser !== null) {
      await browser.close();
    }
  }

  return callback(null, {
    executablePath: await chromium.executablePath,
    headless: chromium.headless,
    version: process.version,
  });
};
```

**Comment by Yilong94** [comment]
I faced a similar issue and it was because I had set the environment variable `IS_OFFLINE` to a non-null string. It seems like this environment variable affects both fields `headless` and `executablePath`. I happened to discover this while looking through the code comments of `chrome-aws-lambda`'s API

```
/**
  * Returns a boolean indicating if we are running on AWS Lambda or Google Cloud Functions.
  * False is returned if Serverless environment variables `IS_LOCAL` or `IS_OFFLINE` are set.
  */
static get headless(): boolean;
``` 

Guess this is not useful anymore but hope it helps anyone who encounters this in the future! Would also be nice if this is documented down in the README  :)
